tests/Satooshi/Bundle/CoverallsV1Bundle/Repository/JobsRepositoryTest.php

```diff
@@ -214,17 +214,16 @@ protected function createResponseMock($statusCode, $reasonPhrase, $body)
         if (is_array($body)) {
             $response
             ->expects($this->once())
-            ->method('json')
-            ->with()
-            ->will($this->returnValue($body));
+            ->method('getBody')
+            ->with(true)
+            ->will($this->returnValue($json));
         } else {
-            $exception = new \Guzzle\Common\Exception\RuntimeException();
-
+            // Error response
             $response
             ->expects($this->once())
-            ->method('json')
-            ->with()
-            ->will($this->throwException($exception));
+            ->method('getBody')
+            ->with(true)
+            ->will($this->returnValue($body));
```

Comment:
isn't the `$json` and `$body` equals here? then the whole `if(is_array($body))` is not needed...